.storybook/rollup.config.js

```diff
@@ -0,0 +1,14 @@
+import mainConfig from '../rollup.config'
+
+let storybookComponentsPackageConfig = Object.assign({}, mainConfig)
+
+storybookComponentsPackageConfig.input = '.storybook/bundle.ts'
+storybookComponentsPackageConfig.output = [
+  {
+    file: 'dist/storybook/components.js',
+    format: 'esm'
+  }
+]
+
+export default storybookComponentsPackageConfig
+
```

Comment:
Actually, the whole assignment could become:

```javascript

export default {
    ...mainConfig,
    input: '.storybook/bundle.ts',
    output: [
        {
            file: 'dist/storybook/components.js',
            format: 'esm'
        }
    ],
};
```

Reply:
again, this is purely a matter of stlye :)

Reply:
And clarity, I strongly agree ;)